tests/cython/openpsi/openpsi_test.py

```diff
@@ -16,22 +18,24 @@ def eat_apple(apple):
     return ConceptNode("finished")
 
 
+__main__.eat_apple = eat_apple
```

Comment:
Is this necessary to make Scheme see `eat_apple`?

Reply:
Yes. Here is the error message which is printed when the eat_apple is not defined in the main module:
`In procedure psi-imply: Python function 'eat_apple' not found in module '__main__'!`